Notation.Plugin.AzureKeyVault/Command/GenerateSignature.cs

```diff
@@ -0,0 +1,59 @@
+using System.Text.Json;
+using Notation.Plugin.AzureKeyVault.Certificate;
+using Notation.Plugin.AzureKeyVault.Client;
+using Notation.Plugin.Protocol;
+
+namespace Notation.Plugin.AzureKeyVault.Command
+{
+    /// <summary>
+    /// Implementation of describe-key command.
+    /// </summary>
+    public class GenerateSignature : IPluginCommand
+    {
+        public async Task<object> RunAsync(string inputJson)
+        {
+            // Parse the input
+            GenerateSignatureRequest? input = JsonSerializer.Deserialize<GenerateSignatureRequest>(inputJson);
+            if (input == null)
+            {
+                throw new ValidationException("Invalid input");
+            }
+
+            var akvClient = new KeyVaultClient(input.KeyId);
+
+            // Extract signature algorithm from the certificate
+            var leafCert = await akvClient.GetCertificate();
+            var keySpec = KeySpecUtils.ExtractKeySpec(leafCert);
+            var signatureAlgorithm = SignatureAlgorithmHelper.FromKeySpec(keySpec);
+
+            // Sign
+            var signature = await akvClient.Sign(signatureAlgorithm, input.Payload);
+
+            // Build the certificate chain
+            List<byte[]> certificateChain = new List<byte[]>();
+            if (input.PluginConfig != null && input.PluginConfig.ContainsKey("ca_certs"))
```

Comment:
Curious if the sugar `input.PluginConfig?.ContainsKey("ca_certs")` works or not.

Reply:
Updated to use `input.PluginConfig?.ContainsKey("ca_certs") ?? false` due to `input.PluginConfig?.ContainsKey("ca_certs")` is also an nullable value.
Thanks!